src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/TabPageTests.cs

```diff
@@ -528,31 +529,91 @@ public void TabPage_AutoSizeMode_SetWithHandleWithParent_GetReturnsExpected(Auto
             Assert.Equal(0, parentLayoutCallCount);
         }
 
-        public static IEnumerable<object[]> BackColor_Get_TestData()
+        [WinFormsTheory]
+        [CommonMemberData(nameof(CommonTestHelper.GetBoolTheoryData))]
+        public static void TabPage_BackColor_Get_ReturnsExpected(bool useVisualStyleBackColor)
         {
-            yield return new object[] { true, null, Control.DefaultBackColor };
-            yield return new object[] { false, null, Control.DefaultBackColor };
+            using var control = new TabPage
+            {
+                UseVisualStyleBackColor = useVisualStyleBackColor
+            };
+            Assert.Equal(Control.DefaultBackColor, control.BackColor);
+        }
 
-            yield return new object[] { true, new TabControl { Appearance = TabAppearance.Normal }, Application.RenderWithVisualStyles ? Color.Transparent : Control.DefaultBackColor };
-            yield return new object[] { false, new TabControl { Appearance = TabAppearance.Normal }, Control.DefaultBackColor };
+        [WinFormsTheory]
```

Comment:
Right now tests that depend on RemoteExecutor doen't work. Please see #3116.
```suggestion
        [WinFormsTheory(Skip = "Crash with AbandonedMutexException. See: https://github.com/dotnet/arcade/issues/5325")]
```